config/locales/en.yml

```diff
@@ -4165,6 +4165,7 @@ en:
   settings: Settings
   share: Share
   share_colon: "Share:"
+  shift: Shift
```

Comment:
Let's change this to

```
# Button label to offset the date/time of observations by a set number of hours
offset_time_verb: Offset
```

That way we're using the same "Offset" verb we're using in the accordion label, translators know it's a verb and not a noun, and they have some understanding of what's being offset.